lib/_sendEvent.ts

```diff
@@ -1,37 +1,79 @@
-import { InsightsSearchClickEvent } from './click';
-import { InsightsSearchConversionEvent } from './conversion';
+import { isNumber, isUndefined, isString, isFunction } from "./utils/index";
 
 declare var process: {
   env: {
     NODE_ENV: string;
   };
 };
 
-export type InsightsEventTypes = 'click' | 'conversion';
+export type InsightsEventType = "click" | "conversion";
```

Comment:
Why we dropped the plural form?

Reply:
@Haroenv suggested to change it here: https://github.com/algolia/search-insights.js/pull/19#discussion_r237392498

What's the best practice for naming enum types?

Reply:
Good question, I quickly search but didn't find anything.

Reply:
We can keep it singular like every other type